user/views.py

```diff
@@ -18,7 +18,6 @@ class CreateTokenView(ObtainAuthToken):
 
```

Comment:
Do you use CreateTokenView anywhere? Because you have to use JWT, so you are free to remove this view and imports from this file